snowshu/adapters/target_adapters/postgres_adapter/postgres_adapter.py

```diff
@@ -313,6 +314,13 @@ def statement_runner(statement: str):
 
     def copy_replica_data(self) -> Tuple[bool, str]:
         status = self.container.exec_run(f"/bin/bash -c '{self.DOCKER_SHARE_REPLICA_DATA}'", tty=True)
+        if self.passive_container:
+            self.container.stop()
+            self.passive_container.start()
+            logger.info('Copying replica data into passive container')
+            self.passive_container.exec_run(f"/bin/bash -c '{self.DOCKER_IMPORT_REPLICA_DATA_FROM_SHARE}'", tty=True)
+            self.passive_container.stop()
+            self.container.start()
```

Comment:
We cannot have both online at the same time due to them occupying same ports, hence the start/stop in multiple places. We need to restart the main container because you cannot commit the stopped one (I guess, might not be true), and the code I made in general assumes that active is running and passive is stopped.